### PR TITLE
added esConsumes to modules in RecoMET/METProducers

### PR DESCRIPTION
**Pull request by JamminJones**
#### PR description:

added esConsumes to 7 modules
#### PR validation:

the code compiles

### CODE REVIEW

**Comment by Dr15Jones** [comment]
The code was changed under my supervision.

**Comment by Dr15Jones** [comment]
please test

File: RecoMET/METProducers/src/HcalNoiseInfoProducer.cc [hunk 1]
```diff
@@ -13,15 +13,9 @@
...
-#include "CalibFormats/HcalObjects/interface/HcalDbService.h"
-#include "CalibFormats/HcalObjects/interface/HcalDbRecord.h"
-#include "RecoLocalCalo/HcalRecAlgos/interface/HcalSeverityLevelComputer.h"
-#include "RecoLocalCalo/HcalRecAlgos/interface/HcalSeverityLevelComputerRcd.h"
 #include "DataFormats/METReco/interface/HcalCaloFlagLabels.h"
 #include "DataFormats/HcalRecHit/interface/HBHERecHitAuxSetter.h"
 #include "DataFormats/HcalRecHit/interface/CaloRecHitAuxSetter.h"
-#include "Geometry/CaloGeometry/interface/CaloGeometry.h"
-#include "Geometry/Records/interface/CaloGeometryRecord.h"
 #include "Geometry/Records/interface/HcalRecNumberingRecord.h"
```
Comment (perrotta) [inline RecoMET/METProducers/src/HcalNoiseInfoProducer.cc:19]: Already in the header

**Comment by perrotta** [comment]
@JamminJones please let us know whether you plan to fix the duplicate inclusion of HcalRecNumberingRecord.h, reported in https://github.com/cms-sw/cmssw/pull/30562#pullrequestreview-443364537

**Comment by JamminJones** [comment]
@perrotta  I am sorry that I did not see this until now but I will fix the duplicate inclusion right away

**Comment by Dr15Jones** [comment]
please test

**Comment by perrotta** [comment]
+1
- Technical
- Jenkinst tests pass

**Comment by silviodonato** [comment]
+1

**Pull request merged**